src/sqlfluff/dialects/dialect_exasol.py

```diff
@@ -3657,15 +3657,15 @@ class CreateAdapterScriptStatementSegment(BaseSegment):
         Sequence(
             "CREATE",
             Ref("OrReplaceGrammar", optional=True),
-            OneOf("JAVA", "PYTHON", Ref("SingleIdentifierGrammar"), optional=True),
+            OneOf("JAVA", "PYTHON", "LUA", Ref("SingleIdentifierGrammar")),
             "ADAPTER",
             "SCRIPT",
         )
     )
     parse_grammar = Sequence(
```

Comment:
Similar to my comment on other PR, should we merge these?

A separate `match_grammar` section should be added when there’s an easy start and terminator clause, to reduce the amount that needs to be parsed. But if it just repeats the first part of the `parse_grammar` then it doesn’t do anything and this change shows that it adds work, when you want to make changes.